sdk/eventhub/event-hubs/README.md

```diff
@@ -90,8 +102,15 @@ Because partitions are owned by the Event Hub, their names are assigned at the t
 To understand what partitions are available, you query the Event Hub using the client.
 
 ```javascript
+const { EventHubClient } = require("@azure/event-hubs");
+
 const client = new EventHubCLient("connectionString", "eventHubName");
-const partitionIds = await client.getPartitionIds();
+
```

Comment:
Should we move the `client` assignment inside the main() as well?

Reply:
On second thoughts lets add `await client.close()` to all samples that do **something** with the client. Else, I believe the program doesn't really exit